src/login/logind-session.c

```diff
@@ -656,6 +657,13 @@ static int session_start_scope(Session *s, sd_bus_message *properties, sd_bus_er
 
                 description = strjoina("Session ", s->id, " of User ", s->user->user_record->user_name);
 
+                after = strv_new("systemd-logind.service",
+                                 s->user->runtime_dir_service,
+                                 s->user->user_record->uid > 0 ? "systemd-user-sessions.service" : STRV_IGNORE,
```

Comment:
hmm, i have the suspicion this should actually be !uid_is_system() or so?

Reply:
I'm not sure about that one too but it might be a sensible change: system services which create session (such as gdm) can be started earlier and basically don't need to wait for /home to be mounted.